.github/workflows/python-build-test.yaml

```diff
@@ -34,12 +34,17 @@ jobs:
           pip install --upgrade pip
           pip install -r requirements-lint.txt
 
-      - name: Lint Python
-        run: |
-          black --check --diff .
-          blackdoc --check --diff .
-          ruff --exit-non-zero-on-fix .
-          vermin --target=3.8- --no-tips --violations .
+      - name: Check format (black)
+        run: black --check --diff .
+
+      - name: Check documentation format (blackdoc)
+        run: blackdoc --check --diff .
+
+      - name: Lint Check (ruff)
+        run: ruff --exit-non-zero-on-fix .
```

Comment:
One thing that occurred to me as I was looking at other stuff: we should change this to

```
ruff --diff .
```

That will make it clearer that something DID fail and why